extension/src/project/projectCommands.ts

```diff
@@ -212,6 +213,16 @@ export function registerProjectCommands(context: vscode.ExtensionContext) {
             }
         }));
 
+        context.subscriptions.push(vscode.commands.registerCommand('autolisp.openWebHelp', () => {
```

Comment:
I think this is not a command use by the "project: feature, it is only used in right click menu.
I suggest we move this function to other files, for example the same file for the "load Lisp" right click menu.

Reply:
Should the openWebHelp.ts move out of the 'project' folder too, or just its registration?

Another reason I liked the notion of it being at the end of the call to registerProjectCommands was that its nearly the very last thing that loads. So even if somebody somewhere does experience problems with my addition, the majority of the extension would be fully loaded without error. I just want you to realize your telling me to move it ahead of a lot of things in the extension.ts.

![image](https://user-images.githubusercontent.com/51800232/93659762-b65fdf80-f9fd-11ea-9132-1b32a979ba81.png)

Reply:
I mean we need to move the whole openWebHelp.ts out of the 'project' folder. And I also suggest to move the register codes out of the projects commands (it is not project command).

maybe put your registration as step 6?

Reply:
Okay, if you tell me the name of the folder the openWebHelp.ts should go in, I can do this. Should it be in its own folder called menus? I think that might make sense due to the other enhancement requests. How else could we "select a symbol and localize it within a defun"? IE, I suspect there will be more context menu commands coming....

Reply:
@JD-Howard I'd say you should create a `extension\src\help` folder and put all your source files there.